quantum/template/ps2avrgb/config.h

```diff
@@ -21,17 +21,19 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define VENDOR_ID       0x20A0
 #define PRODUCT_ID      0x422D
+#define DEVICE_VER      0x0001
 #define MANUFACTURER    You
 #define PRODUCT         %KEYBOARD%
+#define DESCRIPTION     A custom keyboard
 
 #define RGBLED_NUM 16
 
 #define MATRIX_ROWS 2
 #define MATRIX_COLS 3
 
-#define MATRIX_ROW_PINS { D0, D5 }
-#define MATRIX_COL_PINS { F1, F0, B0 }
-#define UNUSED_PINS
+#define MATRIX_ROW_PINS { B0, B1, B2, B3, B4, B5, B6, B7 }
+#define MATRIX_COL_PINS { A0, A1, A2, A3, A4, A5, A6, A7, C7, C6, C5, C4, C3, C2, C1, C0, D7 }
```

Comment:
Due to some vendors not asking for licensing from winkeyless.kr, these pins are MOSTLY correct, but it does not apply to all BMC boards in existence. 

I would terminate the matrix columns at C6

```suggestion
#define MATRIX_COL_PINS { A0, A1, A2, A3, A4, A5, A6, A7, C7, C6 }
```

Reply:
That's a material change from the old version of the template which was hardcoded to scan those pins in that order. I'm a bit leery of just dropping them.

In any case, there's no way to cover all the boards if there are multiple configurations out there, so it's always going to just be a starting point for a port.

Reply:
You are correct. I am simply in favor of dropping them because it adheres to a default template of a 60% BMC board.

Reply:
I'm happy to just put the full list in a comment for reference.

Reply:
That is a good idea!